metaversefile-api.js

```diff
@@ -479,6 +479,7 @@ metaversefile.setApi({
       if (localPlayer.avatar) {
         wasDecapitated = localPlayer.avatar.decapitated;
         localPlayer.avatar.undecapitate();
+        localPlayer.avatar.skeleton.update();

```

Comment:
cause if this will be missing skeleton won't be updated for second render in the mirror.
as a result players head will be missing in the reflection.

Reply:
Before you reverted same PR of other dev BECAUSE of the issue with missing head in the reflection.
Now you say it's ok to make this PR without this fix.
Plz be consistent in your requests, this can continue eternally.

Reply:
> Now you say it's ok to make this PR without this fix.

I didn't say that...

Reply:
so what exactly you suggest ?
this is required thing, maybe can be moved to .undecapitate method,
but same was in neighbour code about the opposite and same structure so I did same.

Reply:
So ?